tldw_Server_API/app/api/v1/endpoints/prototype_workspaces.py

```
@@ -0,0 +1,423 @@
+
+
+@router.post(
+    "/prototype-previews/{preview_handle}/renew",
+    response_model=PrototypePreviewGrantResponse,
+    status_code=status.HTTP_200_OK,
+    summary="Renew a prototype preview grant",
+)
+async def renew_preview_grant(
+    preview_handle: str,
+    _body: PrototypePreviewRenewRequest,
+    user: User = Depends(get_request_user),
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)